### PR TITLE
Softmax reward model outputs

### PR DESCRIPTION
**Pull request by opentaco**
Logits appear to be the outputs of both open_assistant and reciprocate models, but then requires softmax treatment for correct interpretation. Current standardization can then be removed, due to softmax itself just having a relative interpretation and always summing to 1. This change is intended to improve the interpretation of these reward model outputs.

### CODE REVIEW

**Comment by opentaco** [comment]
Softmax will, however, inappropriately amplify poor rewards via its normalization, in the absence of good rewards in the batch. The logit skew introduced without softmax have not sullied the reward interpretation noticeably yet, so will be retained for now.
